master/static/srv/get_model_versions.sql

```diff
@@ -10,13 +10,14 @@ WITH mv AS (
     labels,
     notes,
     username,
+    user_id,
     last_updated_time
   FROM model_versions
   LEFT JOIN users ON users.id = model_versions.user_id
   WHERE model_id = $1
 ),
 m AS (
-  SELECT m.id, m.name, m.description, m.notes, m.metadata, m.creation_time, m.last_updated_time, array_to_json(m.labels) AS labels, u.username, m.archived, COUNT(mv.version) as num_versions
+  SELECT m.id, m.name, m.description, m.notes, m.metadata, m.creation_time, m.last_updated_time, array_to_json(m.labels) AS labels, u.username, m.user_id, m.archived, COUNT(mv.version) as num_versions
```

Comment:
this is a problem in the existing master branch too, but some fields (username, user_id) are SELECT-ed properly here but also need to be added to the SELECT at the end of this query.  If it is missing you get `"username": "", "userId": 0,` back